test/conftest.py

```diff
@@ -1,8 +1,7 @@
 import os
-
-import pytest
 from test import RunFromText
```

Comment:
isort should be made aware of the first-party test helpers. It's best to use an explicit relative import for this.